docs/machine-learning/how-to-guides/load-data-model-builder.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Set up & load data for Model Builder
+description: Learn how to set up and load data from a SQL Server database or a file for use in one of the Model Builder scenarios for ML.NET.
+ms.date: 10/31/2019
+author: luisquintanilla
+ms.author: luquinta
+ms.custom: mvc, how-to
+#Customer intent: As a developer, I want to load data in Model Builder
+---
+
+# Load data in Model Builder
+
+Learn how to set up and load your data sets from a SQL Server or a file database for use in one of the Model Builder scenarios for ML.NET. Model Builder scenarios can use SQL files, image files, and CSV or TSV file formats.
+
+## Data set limitations in Model Builder
+
+Model Builder limits the amount of data you can use for training models:
+
+- Training data set limit: 1GB
+- SQL Server data: 100,000 rows 
+
+[Question: Clarify the limits. Is the SQL Server data limit an absolute limit or for training only? The training dataset  -- is this a limit for text or numerical data from files?]
```

Comment:
The SQL Server data limit of 100k rows is only for training. Once a model is trained, a user can make predictions on as many rows as they need to using the trained model because inferencing is done outside of Model Builder in the users's application.